src/js/app.js

```diff
@@ -3,9 +3,9 @@
  *
  * This is where you write your app.
  */
-
-var UI = require('ui');
-var Vector2 = require('vector2');
+require('./main');
```

Comment:
This is possible if you use a custom wscript file in Pebble.js apps (w/changes to CP), otherwise not. This will not be possible if we intend to make Pebble.js a standard Pebble Package.